### ISSUE TITLE
Blue background color and select color are too similar

### ISSUE THREAD

**Issue by felixbinder** [zoxon/vscode-theme-monokai-light#5]
VS Code 1.57.1, macOS Big Sur

The text of partial search matches in the top menu is the same as the highlight color, making this hard to parse.

![image](https://user-images.githubusercontent.com/24477285/123431611-be226500-d5c9-11eb-8677-0cad28b6d73f.png)
![image](https://user-images.githubusercontent.com/24477285/123431617-bfec2880-d5c9-11eb-9498-bc4eeafeb525.png)

**Comment by zoxon** [comment]
Duplicates #4
